### PR TITLE
【translate】mds/nameserver2

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/mds/nameserver2/curvefs.cpp [hunk 1]
```diff
@@ -816,7 +819,7 @@ StatusCode CurveFS::RenameFile(const std::string & oldFileName,
         }
         return StatusCode::kOK;
     } else if (ret3 == StatusCode::kFileNotExists) {
-        // newFileName不存在, 直接rename
+        // newFileName does not exist, renamevdirectly
```
Comment (wu-hanqing) [inline src/mds/nameserver2/curvefs.cpp:822]: rename directly

File: src/mds/nameserver2/allocstatistic/alloc_statistic.cpp [hunk 2]
```diff
@@ -89,17 +89,17 @@ bool AllocStatistic::GetAllocByLogicalPool(PoolIdType lid, int64_t *alloc) {
 
 void AllocStatistic::AllocSpace(
     PoolIdType lid, int64_t changeSize, int64_t revision) {
-    // segmentAlloc_值不可用,change需要更新到existSegmentAllocValues_中
+    // segmentAlloc_ value is not available, change needs to be updated to existSegmentAllocValues_ //NOLINT
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.cpp:92]: > chang改成 `changeSize'

done

File: src/mds/nameserver2/allocstatistic/alloc_statistic.cpp [hunk 3]
```diff
@@ -89,17 +89,17 @@ bool AllocStatistic::GetAllocByLogicalPool(PoolIdType lid, int64_t *alloc) {
...
         existSegmentAllocValues_[lid] += changeSize;
     }
 
-    // 如果etcd中的数据已经统计结束，直接将change更新到segmentAlloc_中
+    // update change to segmentAlloc_ directly if the Etcd data has been counted
     if (true == segmentAllocFromEtcdOK_.load()) {
         WriteLockGuard guard(segmentAllocLock_);
         segmentAlloc_[lid] += changeSize;
-    // 如果etcd中的数据还未统计结束，将change更新到segmentChange_中
+    // if the Etcd data has not been counted, update change to segmentChange_
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.cpp:102]: > 这个和上面的也是

done

File: src/mds/nameserver2/allocstatistic/alloc_statistic.cpp [hunk 4]
```diff
@@ -132,36 +132,36 @@ void AllocStatistic::CalculateSegmentAlloc() {
 
     LOG(INFO) << "calculate segment alloc revision not bigger than "
               << curRevision_ << " ok";
-    // 设置从etcd中获取数据完成
+    // set to get data from etcd
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.cpp:135]: > set fetch data from etcd success ?

done

File: src/mds/nameserver2/allocstatistic/alloc_statistic.cpp [hunk 5]
```diff
@@ -242,19 +241,20 @@ std::map<PoolIdType, int64_t> AllocStatistic::GetLatestSegmentAllocInfo() {
...
     int64_t sumChangeUntilNow = 0;
 
     {
         WriteLockGuard guard(segmentChangeLock_);
-        // 获取指定lid对应的map
+        // get the map corresponding to the specified lid
         auto liter = segmentChange_.find(lid);
         if (liter != segmentChange_.end()) {
-            // change中的没有数据，可以直接删除，后续直接更新到segment中
+            // there's no data in the change, delete directly
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.cpp:252]: > change -> segmentChange_

done

File: src/mds/nameserver2/allocstatistic/alloc_statistic.cpp [hunk 6]
```diff
@@ -242,19 +241,20 @@ std::map<PoolIdType, int64_t> AllocStatistic::GetLatestSegmentAllocInfo() {
...
 
 void AllocStatistic::UpdateSegmentAllocByCurrrevision(PoolIdType lid) {
-    // 获取>revision之后的值
+    // sum up the value after the revision
     int64_t sumChangeUntilNow = 0;
 
     {
         WriteLockGuard guard(segmentChangeLock_);
-        // 获取指定lid对应的map
+        // get the map corresponding to the specified lid
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.cpp:249]: > 这里的lid指的是逻辑池还是物理池？

逻辑池

File: src/mds/nameserver2/allocstatistic/alloc_statistic.h [hunk 7]
```diff
@@ -45,32 +45,35 @@ namespace mds {
...
- * 思路
- * 统计分为两部分:
+ * AllocStatistic is for counting the number of segments allocated currently
+ * The statistics are divided into two parts:
  * part1:
- *    ①统计指定revision之前的分配量
- *    ②记录mds启动以来，每个revision对应的segment分配量
- *    ③合并①和②中的数据
- * part2: 后台定期持久化part1中合并以后的数据
+ *     ①statistics of the allocation amount before the designated revision
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.h:51]: > 这里的1、2、3直接用1/2/3表示吧

done

File: src/mds/nameserver2/allocstatistic/alloc_statistic.h [hunk 8]
```diff
@@ -86,142 +89,151 @@ class AllocStatistic {
...
+     * @brief CalculateSegmentAlloc Get all the segment records of the
+     *                         specified revision from Etcd
      */
     void CalculateSegmentAlloc();
 
     /**
-     * @brief PeriodicPersist
-     *        定期持久化内存中的统计的各logcailPool下的已分配的segment大小
+     * @brief PeriodicPersist Periodically persist the allocated segment size
+     *                        data under each logicalPool in RAM
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic.h:153]: > 上面一处内存用的是memory，这里用的是RAM，统一一下吧

done, 统一成memory了

File: src/mds/nameserver2/allocstatistic/alloc_statistic_helper.cpp [hunk 9]
```diff
@@ -52,7 +50,7 @@ int AllocStatisticHelper::GetExistSegmentAllocValues(
         return -1;
     }
 
-    // 解析
+    // recode the string
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply src/mds/nameserver2/allocstatistic/alloc_statistic_helper.cpp:53]: > decode the string ?

done

File: src/mds/nameserver2/curvefs.cpp (hunk 1 above)
Comment (SeanHai) [reply src/mds/nameserver2/curvefs.cpp:822]: > rename directly

done

**Review by wu-hanqing — APPROVED** [review]

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**